### PR TITLE
my stats

### PR DESCRIPTION
**Pull request by askarjon111**

### CODE REVIEW

File: src/data/messages.py [hunk 1]
```diff
@@ -1,68 +1,69 @@
-msg_dict = {
```
Comment (DavraYoung) [inline src/data/messages.py:1]: Why this change exist? Fix the line endings  and force push

File: src/main.py [hunk 2]
```diff
@@ -1,55 +1,55 @@
-import asyncio
```
Comment (DavraYoung) [inline src/main.py:1]: why the whole file is changed?

**Review by DavraYoung — CHANGES_REQUESTED** [review]

File: src/data/messages.py (hunk 1 above)
Comment (askarjon111) [reply src/data/messages.py:1]: force push to main?
